template.tpl

```diff
@@ -0,0 +1,2413 @@
+  },
+  "containerContexts": [
+    "WEB"
+  ]
+}
+
+
+___TEMPLATE_PARAMETERS___
+
+[
+  {
+    "type": "SELECT",
+    "name": "ecomAPI",
+    "displayName": "Ecommerce API",
+    "macrosInSelect": false,
+    "selectItems": [
+      {
+        "value": "spEcommerce",
+        "displayValue": "Snowplow Ecommerce"
+      },
+      {
+        "value": "ga4Ecommerce",
+        "displayValue": "Google Analytics 4 Ecommerce"
+      },
+      {
+        "value": "uaEcommerce",
+        "displayValue": "Universal Analytics Enhanced Ecommerce"
+      }
+    ],
+    "simpleValueType": true,
+    "defaultValue": "spEcommerce",
+    "help": "Specify the Snowplow Ecommerce API to use. If you already use Google Analytics 4 ecommerce or Universal Analytics Enhanced Ecommerce to collect information about the shopping behavior of your users, you can use the respective APIs as a way to quickly implement Snowplow Ecommerce without making many changes on your current setup. However to get the full value of it, you should properly plan a full migration towards Snowplow Ecommerce API.",
+    "alwaysInSummary": true,
+    "valueValidators": [
+      {
+        "type": "NON_EMPTY"
+      }
+    ]
+  },
+  {
+    "type": "GROUP",
+    "name": "snowplowGroup",
+    "displayName": "Tracking Parameters",
+    "groupStyle": "ZIPPY_OPEN",
+    "subParams": [
+      {
+        "type": "SELECT",
+        "name": "spEcomFunction",
+        "displayName": "Ecommerce Function",
+        "macrosInSelect": false,
```

Comment:
Particularly if you've got a single ecommerce Variable it might be good to allow macros here so people can just map their events with a lookup table, rather than needing to copy the tag a bunch of times for each event.